src/open_petro_elastic/material/span_wagner/carbon_dioxide.py

```diff
@@ -0,0 +1,551 @@
+    ...     print_expr(f's{i+1}', s, True)
+    ... print(sep)
+    ...
+    ... # First derivatives
+    ... print_expr('dt_theta', theta_expr.diff(tau), False)
+    ... print_expr('dt_bigdelta', bigdelta_expr.diff(tau), False)
+    ... print_expr('dt_bigphi', bigphi_expr.diff(tau), False)
+    ... for i, s in enumerate([s1, s2, s3, s4]):
+    ...     print_expr(f'dt_s{i+1}', s.diff(tau), True)
+    ... print(sep)
+    ... print_expr('dd_theta', theta_expr.diff(delta), False)
+    ... print_expr('dd_bigdelta', bigdelta_expr.diff(delta), False)
+    ... print_expr('dd_bigphi', bigphi_expr.diff(delta), False)
+    ... for i, s in enumerate([s1, s2, s3, s4]):
+    ...     print_expr(f'dd_s{i+1}', s.diff(delta), True)
+    ... print(sep)
+    ...
+    ... # Second derivatives
+    ... print_expr('dtt_theta', theta_expr.diff(tau, tau), False)
+    ... print_expr('dtt_bigdelta', bigdelta_expr.diff(tau, tau), False)
+    ... print_expr('dtt_bigphi', bigphi_expr.diff(tau, tau), False)
+    ... for i, s in enumerate([s1, s2, s3, s4]):
+    ...     print_expr(f'dtt_s{i+1}', s.diff(tau, tau), True)
+    ... print(sep)
+    ... print_expr('ddd_theta', theta_expr.diff(delta, delta), False)
+    ... print_expr('ddd_bigdelta', bigdelta_expr.diff(delta, delta), False)
+    ... print_expr('ddd_bigphi', bigphi_expr.diff(delta, delta), False)
+    ... for i, s in enumerate([s1, s2, s3, s4]):
+    ...     print_expr(f'ddd_s{i+1}', s.diff(delta, delta), True)
+    ... print(sep)
+    ... print_expr('ddt_theta', theta_expr.diff(delta, tau), False)
+    ... print_expr('ddt_bigdelta', bigdelta_expr.diff(delta, tau), False)
+    ... print_expr('ddt_bigphi', bigphi_expr.diff(delta, tau), False)
+    ... for i, s in enumerate([s1, s2, s3, s4]):
+    ...     print_expr(f'ddt_s{i+1}', s.diff(delta, tau), True)
+    ... print(sep)
+
+The expressions are directly equivalent to the expressions in the code below, with an exception being sympy's DiracDelta
+function. This can, however, be directly replaced with a boolean expression.
+"""
+from numpy import array, log, exp, sum, abs, sign
+from open_petro_elastic import float_vectorize
+
+from open_petro_elastic.material.fluid import fluid_material as fluid
+from open_petro_elastic.material.conversions import celsius_to_kelvin
+
+
+"""
+TODO:
+    - Handle density (and other derived properties) close to phase transitions, critical point and triple point
+    - Support vectorized variants of the below functions
+"""
+
+
+# Constants
+R = 0.1889241 * 1e3  # J / kg K
+T_critical = 304.1282  # K
+rho_critical = 467.6  # kg / m^3
+p_critical = 7.3773  # MPa
+T_triple = 216.592  # K
+p_triple = 0.51795  # MPa
+
+
+"""
+Span-Wagner Coefficients
+
+Coefficients defined in Table 31 of Span & Wagner. Coefficients are divided into groups according to horizontal lines in
+the table. This is also convenient when evaluating the Helmholtz energy expressions (phi functions).
+"""
+
+
+""" Ideal gas part """
```

Comment:
Probably, its a good idea to keep the tables in csv files.  If you write the tables into
`src/open_petro_elastic/material/span_wagner/csv_tables/table_31.csv` . And add a file
`src/open_petro_elastic/material/span_wagner/tables.py` containing 
```
import pandas as pd
table_31 = pd.read_scv(
  path.join(path.dirname(__file__), "csv_tables", "table_31.csv")
)
```
Alternatively just keep the table values in a different file.

Reply:
Moved the coefficients to a separate Python file for now: `src/open_petro_elastic/material/span_wagner/coefficients.py`. Due to varying lengths on the arrays, csv is not very convenient. Other formats are perhaps suited, but for now, having them in a separate Python file at least helps clean up a bit.